canonical-web-and-design/jaas-dashboard
# Refactor to move all panels to top level src

## Done

- Refactor to move all panels to top level src
- Create `tableHeaders.js` and `tableRows.js`

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Make sure everything runs as expected
